src/modules/search/components/menu/types/geoResource/GeoResourceResultItem.js

```diff
@@ -0,0 +1,112 @@
+import { html, nothing } from 'lit-html';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+import { $injector } from '../../../../../../injection';
+import { BaElement } from '../../../../../BaElement';
+import { addLayer, removeLayer } from '../../../../../../store/layers/layers.action';
+import { MainMenuTabIndex } from '../../../../../menu/components/mainMenu/MainMenu';
+import { close as closeMainMenu, setTabIndex } from '../../../../../menu/store/mainMenu.action';
+import itemCss from '../item.css';
+import css from './geoResourceResultItem.css';
+
+
+
+/**
+ * Renders an search result item for a geoResource.
+ * 
+ * Configurable Properties:
+ * - `data`
+ * 
+ * Observed Properties:
+ * - `data`
+ * 
+ * @class
+ * @author taulinger
+ */
+export class GeoResourceResultItem extends BaElement {
+	
+	constructor() {
+		super();
+
+		const { EnvironmentService } = $injector.inject('EnvironmentService');
+		this._environmentService = EnvironmentService;
+		this._portrait = false;
+	}
+
+
+	initialize() {
+
+		const _window = this._environmentService.getWindow();
+		//MediaQuery for 'orientation'
+		const mediaQuery = _window.matchMedia('(orientation: portrait)');
+		const handleOrientationChange = (e) => {
+			this._portrait = e.matches;
+		};
+		mediaQuery.addEventListener('change',  handleOrientationChange);
+		//initial set of local state
+		handleOrientationChange(mediaQuery);
+	}
+
+	set data(geoResourceSearchResult) {
+		this._georesourceSearchResult = geoResourceSearchResult;
+		this.render();
+	}
+
+	static _tmpLayerId(id) {
+		return `tmp_${GeoResourceResultItem.name}_${id}`;
+	}
+
+	createView() {
+		/**
+		 * Uses mouseenter and mouseleave events for adding/removing a preview layer.
+		 * These events are not fired on touch devices, so there's no extra handling needed.
+		 */
+		const onMouseEnter = (result) => {
+			//add a preview layer
+			addLayer(GeoResourceResultItem._tmpLayerId(result.id), { label: result.label, geoResourceId: result.id, constraints: { hidden: true, alwaysTop: true } });
+		};
+		const onMouseLeave = (result) => {
+			//remove the preview layer
```

Comment:
a more philosophical question of our coding style, then a real proposal for change, which @alsturm and i stumbled over in the last changes of the layermanager...

Do we prefer smallest helper methods and expressions with a meaning name, instead of the pure implementation with a meaningful comment.

In this case:
`const removePreviewLayer = () => removeLayer(GeoResourceResultItem._tmpLayerId(result.id));`
instead of:
`//remove the preview layer` 
`removeLayer(GeoResourceResultItem._tmpLayerId(result.id));`

Reply:
I don't think we should use  a helper method just to make clear what's going on, instead we  can use a meaningful name when we import the action 
`import { foo as bar } ... `

Reply:
Hm, we should 'allow' both approaches, the intention should be clear...